Max-Movies-In-Years-2.js

```diff
@@ -0,0 +1,70 @@
+function displayMap(map) {
+    map.forEach((value, key) => console.log(key + ":" + value))
+}
+
+//TODO: if year array empty , space complexity improved , filling of map need to be seperated to new func call 
+// TODO:  explain need for sort and why wrote that comprator 
+// TODO: wrong space complexity evaluated 
+
+//Response consist of obj ->year [start,end]
+const arr = [{
+        name: "kratika",
+        year: [2008, 2010]
+    },
+    {
+        name: "kratika2",
+        year: [2005, 2012]
+    },
+    {
+        name: "kratika3",
+        year: [2013, 2014]
+    },
+    {
+        name: "kratika4",
+        year: [2007, 2011]
+    }
+]
+
```

Comment:
Any reason why there is `-2` in the name? and a file with `-3`

Remove dead code please